README.rst

```diff
@@ -33,7 +33,7 @@ Installation
 
 .. code-block:: bash
 
-	$ brew install caskroom/cask/wkhtmltopdf
+	$ brew install homebrew/cask-cask/wkhtmltopdf
```

Comment:
```suggestion
	$ brew cask install wkhtmltopdf
```

As outlined in https://github.com/JazzCore/python-pdfkit/issues/149#issuecomment-677507216 brew specifies the correct method as `brew cask install`.



I have also created #168 as I hadn't seen this one but only one is necessary so I can close mine as well.